### PR TITLE
stylelint configration

### PR DESCRIPTION
**Pull request by Nerdy-Girl**

### CODE REVIEW

**Comment by Nerdy-Girl** [comment]
#23

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "",
   "main": "src/components/_base.scss",
   "scripts": {
+    "stylelint": "stylelint **/*.scss **/*.scss.example",
```
Comment (ArekRado) [inline package.json]:
```
"stylelint": "stylelint  /src/**/*.scss /src/**/*.scss.example"
```

**Review by ArekRado — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Nerdy-Girl) [reply package.json]: it's working without /src/ 
😃

**Comment by Nerdy-Girl** [comment]
^ @ArekRado https://github.com/stylelint/stylelint/pull/2464/files#diff-e3c9b75ef08163744e78a99035b50be2R289

**Review by ArekRado — APPROVED** [review]

**Pull request merged**